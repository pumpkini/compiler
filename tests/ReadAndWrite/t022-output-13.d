int main() {
    Print(-4758);
    Print(9802);
    Print(1003);
    Print(-99999);
}
