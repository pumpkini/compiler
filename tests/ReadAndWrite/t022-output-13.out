-4758
9802
1003
-99999
